src/core/libs/FactoryLib.sol

```diff
@@ -80,6 +80,23 @@ library FactoryLib {
         uint len;
     }
 
+    // Mapping from address to alias name
+    mapping(address => string) public aliasNames;
```

Comment:
* this mapping must be in Factory ERC7201 storage
* use named parameters in mapping types (Solidity 0.8.18+)